### PR TITLE
fix: fix a broken diff_test on Windows requiring legacy external runfiles

### PR DESCRIPTION
**Pull request by kormide**

### CODE REVIEW

**Review by gregmagolan — APPROVED** [review]
🌮

File: .bazelrc.common [hunk 1]
```diff
@@ -21,9 +21,16 @@ startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
 #          and developer mode is enabled.
 build --enable_runfiles
 
-# Turn off legacy external runfiles
-# This prevents accidentally depending on this feature, which Bazel will remove.
-build --nolegacy_external_runfiles
+common --enable_platform_specific_config
+
+# Turn off legacy external runfiles on all platforms except Windows.
+# This prevents accidentally dependening on this feature, which Bazel will remove.
```
Comment (alexeagle) [inline .bazelrc.common:27]: nit: spelling

(why is my brain like that?)

**Review by alexeagle — APPROVED** [review]
thanks, this is fine I think

File: .bazelrc.common (hunk 1 above)
Comment (kormide) [reply .bazelrc.common:27]: :scream_cat: I even carefully proofread this..

**Pull request merged**